spirv_cross.cpp

```diff
@@ -1018,6 +1018,9 @@ uint32_t Compiler::get_decoration(uint32_t id, Decoration decoration) const
 		return dec.input_attachment;
 	case DecorationSpecId:
 		return dec.spec_id;
+	case DecorationNonWritable:
+	case DecorationNonReadable:
+		return 1;
 	default:
```

Comment:
The default case should just return 1 here.
This needs to be implemented for get_member_decoration as well.
The header should be updated to mention this new behavior.